docs/guide/explanation/the-deployment-process.mdx

```diff
@@ -0,0 +1,53 @@
+# The deployment process
+Deployment begins via one of Zephyr Cloud's bundler plugins.
+When added to your bundler's plugin list, the plugin deploys your bundled applicatioon to a remote Zephyr Cloud instance.
+
+## Deployment lifecycle
+The deployment lifecycle can be split into three stages:
+
+*   [Initial bundling](#initial-bundling)
+*   [Fact gathering](#fact-gathering)
+*   [Snapshot upload and tagging](#snapshot-upload-and-tagging)
+
+### Initial bundling
+The application is bundled according to your bundler's configuration.
+The Zephyr Cloud bundler plugin runs as the last step in the bundling process, initiating the next stage of the process.
+
+### Fact gathering
+The bundler plugin first attempts to gather facts about the application.
+Deployments are identified by metadata relating to the application, including:
+
+*   Details extracted to the Git repository's `origin` remote URL:
+    *   The username/organization
+    *   The repository name
+*   The current Git branch
+*   [Whether the application has Module Federation enabled](#module-federation-and-micro-frontends)
+
+Once the details have been gathered, the plugin establishes a connection to the remote Zephyr Cloud instance.
+If not already authenticated, the plugin opens a browser window to authenticate.
+Otherwise, the plugin uses previously stored credentials located in `~/.zephyr` to authenticate.
+
+### Snapshot upload and tagging
+The snapshot created by the bundler is sent to Zephyr Cloud's API alongside the facts gathered in the previous stage.
```

Comment:
Perhaps we want to make a comment about how the assets are uploaded directly to the cloud provider of their choice rather than to our cloud then their cloud

Reply:
@zackarychapple does this functionality exist today, or is the only option right now having Zephyr Cloud's managed offering host the assets? If so, @zmzlois can you provide more technical context on how someone can configure a custom cloud provider.

Reply:
try + check it at here https://app.zephyr-cloud.io/org/{orgName}/settings/deployment-integrations 

couldn't write much about it since we are going through big redesign of the dashboard so both navigation, naming, content here will change a lot

Reply:
for now I'd rather we don't write about it too much until we are sure about the naming, layout, usage flow on dashboard since it's not clear and still waiting for design